src/libraries/System.Runtime.Serialization.Xml/ref/System.Runtime.Serialization.Xml.cs

```diff
@@ -6,6 +6,50 @@
+        public virtual System.Xml.XmlQualifiedName GetArrayTypeName(bool isNullable) { throw null; }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Data Contract Serialization and Deserialization might require types that cannot be statically analyzed. Make sure all of the required types are preserved.")]
+        public static DataContract? GetBuiltInDataContract(string name, string ns) { throw null; }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Data Contract Serialization and Deserialization might require types that cannot be statically analyzed. Make sure all of the required types are preserved.")]
+        public static DataContract GetDataContract(Type type) { throw null; }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Data Contract Serialization and Deserialization might require types that cannot be statically analyzed. Make sure all of the required types are preserved.")]
+        public static System.Xml.XmlQualifiedName GetStableName(Type type) { throw null; }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Data Contract Serialization and Deserialization might require types that cannot be statically analyzed. Make sure all of the required types are preserved.")]
+        public static Type GetSurrogateType(ISerializationSurrogateProvider surrogateProvider, Type type) { throw null; }
+        public virtual bool HasRoot { get { throw null; } }
+        public virtual bool IsBuiltInDataContract { get { throw null; } }
+        public virtual bool IsISerializable { get { throw null; } }
+        public virtual bool IsKeyValue(out string? keyName, out string? valueName, out string? itemName) { throw null; }
+        public virtual bool IsReference { get { throw null; } }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Data Contract Serialization and Deserialization might require types that cannot be statically analyzed. Make sure all of the required types are preserved.")]
+        public static bool IsTypeSerializable(Type type) { throw null; }
+        public virtual bool IsValueType { get { throw null; } }
+        public virtual System.Collections.Generic.IDictionary<System.Xml.XmlQualifiedName, System.Runtime.Serialization.DataContract>? KnownDataContracts { [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Data Contract Serialization and Deserialization might require types that cannot be statically analyzed. Make sure all of the required types are preserved.")] get { throw null; } set { throw null; } }
+        public virtual System.Collections.Generic.List<DataMember>? Members { get { throw null; } }
+        public virtual Type OriginalUnderlyingType { get { throw null; } }
```

Comment:
Should we make this an `IList<DataMember>` too?

Reply:
Yup. Already on that one as I was getting the API a little more readably formatted for review.

Reply:
In general, properties shouldn't use interfaces of collections but rather concrete collections (`List<T>` instead of `IList<T>`), unless the code (a) returns a non-public type and the interface is the only way to describe it or (b) multiple different types are potentially returned in a default implementation (varying on some piece of context somewhere)